taskcluster/kinds/shortlist/kind.yml

```diff
@@ -43,7 +43,7 @@ tasks:
                 - dependencies
                 - worker.env
                 - attributes
-        worker-type: b-cpu-xlargedisk
+        worker-type: b-cpu-xlargedisk-32-256
```

Comment:
@gabrielBusta shortlist script is using a proper tokenization with sentencespiece so the vocabulary size is normal in this case and I don't expect it to require a lot of memory

Reply:
ah, a bit off-topic, but can the other alignment tasks also use "proper tokenization"?

Reply:
yes, but it's a bit complicated because OpusTrainer expects naive tokenization, so we'll need to fix it there too. See https://github.com/mozilla/firefox-translations-training/issues/507 and https://github.com/hplt-project/OpusTrainer/issues/38